Cargo.toml

```diff
@@ -16,7 +16,8 @@ edition = "2018"
 [package.metadata.docs.rs]
 features = ["std", "num-bigint-std", "serde"]
 
-[dependencies]
+[dependencies.paste]
+version = "1.0"
```

Comment:
Why is this dependency required?

Reply:
its for generating the call to the corresponding underlying impl `::[<from_ $name>]`

```rust
                    fn try_from(n: $name) -> Result<Self, ()> {
                        <$into as FromPrimitive>::[<from_ $name>](n)
                            .map(Ratio::from_integer)
                            .ok_or(())
                    }
```